KianCommons/UI/Table/UITable.cs

```diff
@@ -0,0 +1,83 @@
+using ColossalFramework.UI;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace KianCommons.UI.Table {
+    class UITable : UIPanel {
+        public UITableRow[] rows = new UITableRow[0];
```

Comment:
public fiedls should be CamelCase
private fields should be camelCase_

Also do you really want to expose this field? maybe it should be private with property to access it.

Reply:
changed, now private. Though I might need to add a getter later for styling.

Reply:
added the trailing underscore; mistook that for a typo first.